sdk/maps/maps-render-rest/CHANGELOG.md

```diff
@@ -10,6 +10,8 @@
 
 ### Other Changes
```

Comment:
are you going to have a release of @azure-rest/maps-render ?

Reply:
Maybe move them to Breaking Changes, just make sure remove other sections like Features Added, if this is the only change you want to take in 1.0.0-beta.2

Reply:
@qiaozha Yes there will be a new release for the package. As @dubiety [mentioned previously](https://github.com/Azure/azure-sdk-for-js/pull/26349/files/f026c8d6336df9ffca979356304bd8157c315ea3#r1245332108), C# archetect suggest adding this changes in "Other Changes" since the package is in beta.
But let me know if we have different conventions in TypeScript.

I'll remove other sections, thanks for heads up :)